packages/one-app-bundler/README.md

```diff
@@ -90,9 +90,9 @@ npm install some-dependency
 
 Modules shouldn't configure both `providedExternals` and `requiredExternals`.
```

Comment:
Would it be clearer to discuss this from the start of the section? Looking over the section on externals, it doesn't become apparent what each is used for (root and child module) until the end of the article (scattered between the lines).